src/preload.ts

```diff
@@ -594,7 +595,8 @@ setInterval(function () {
     };
 
     // update title, url and play info with new info
-    setTitle(songDashArtistTitle);
+    if(settingsStore.get(settings.staticWindowTitle)) setTitle(staticTitle)
```

Comment:
Please either write out the entire if or, if you want to use shorthand, you can use a ternary:

```
settingsStore.get(settings.staticWindowTitle) ? setTitle(staticTitle) : setTitle(songDashArtistTitle);
```

Reply:
Changed to ternary on last commit.